fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -389,7 +389,16 @@ public Response createOrReplaceObjectRdf(
         }
 
         try {
-            session.save();
+            // prevent SN-Sibling nodes from creating.
+            synchronized (LOGGER) {
```

Comment:
Yes, it sounds good. I tried it again to push the logic down to FedoraLdp.createFedoraResource, but my test still get failed with the SN-sibling file get created (status code 201 instead of 409). So this should be the last chance to prevent the SN-sibling file from being created in this specific case.